### DISCUSSION TITLE
Issue fetch models folder from other Drive

### DISCUSSION THREAD

**Discussion by chnisar515** [comfyanonymous/ComfyUI#5204] [General]
Hello Experts,
Today i fresh download portable comfyui in my C Drive and i have my models folder saved in my E Drive so now i just Modify my (extra_model_paths.yaml) file like below

> other_ui:
base_path: E:\comfyuimodels\models
checkpoints: models/checkpoints/
clip: models/clip/
clip_interrogator: models/clip_interrogator/
clip_vision: models/clip_vision/
configs: models/configs/
controlnet: models/controlnet/
diffusers: models/diffusers/
diffusion_models: models/diffusion_models/
embeddings: models/embeddings/
gligen: models/gligen/
hypernetworks: models/hypernetworks/
LLM: models/LLM/
llm_gguf: models/llm_gguf/
loras: models/loras/
photomaker: models/photomaker/
style_models: models/style_models/
unet: models/unet/
upscale_models: models/upscale_models/
vae: models/vae/
vae_approx: models/vae_approx/

and now when i try to run (run_nvidia_gpu.bat) then i got this error below

> C:\ComfyUI_windows_portable>.\python_embeded\python.exe -s ComfyUI\main.py --windows-standalone-build
[START] Security scan
[DONE] Security scan
 ComfyUI-Manager: installing dependencies done.
** ComfyUI startup time: 2024-10-10 18:46:41.384160
** Platform: Windows
** Python version: 3.11.9 (tags/v3.11.9:de54cf5, Apr  2 2024, 10:12:12) [MSC v.1938 64 bit (AMD64)]
** Python executable: C:\ComfyUI_windows_portable\python_embeded\python.exe
** ComfyUI Path: C:\ComfyUI_windows_portable\ComfyUI
** Log path: C:\ComfyUI_windows_portable\comfyui.log
Prestartup times for custom nodes:
1.0 seconds: C:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager
Total VRAM 12288 MB, total RAM 32607 MB
pytorch version: 2.4.1+cu124
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 3060 : cudaMallocAsync
Using pytorch cross attention
[Prompt Server] web root: C:\ComfyUI_windows_portable\ComfyUI\web
Traceback (most recent call last):
  File "C:\ComfyUI_windows_portable\ComfyUI\main.py", line 207, in <module>
    utils.extra_config.load_extra_path_config(extra_model_paths_config_path)
  File "C:\ComfyUI_windows_portable\ComfyUI\utils\extra_config.py", line 21, in load_extra_path_config
    for y in conf[x].split("\n"):
             ~~~~^^^
TypeError: string indices must be integers, not 'str'
C:\ComfyUI_windows_portable>pause
Press any key to continue . . .

and when i disable (extra_model_paths.yaml) file i mean when i rename it again to (extra_model_paths.yaml.example) then comfyui running perfectly and generating images.

**Comment by LukeG89** [comment]
I'm not an expert, but I would try replacing `\` with` /` in your `base_path`, like in the other folder paths.

**Reply by chnisar515** [reply]
> I'm not an expert, but I would try replacing `\` with` /` in your `base_path`, like in the other folder paths.

i tried it but the issue still not solve. so now i fixed it with the below.

> other_ui:
    base_path: E:\comfyuimodels
    checkpoints: models/checkpoints/
    clip: models/clip/
    clip_interrogator: models/clip_interrogator/
    clip_vision: models/clip_vision/
    configs: models/configs/
    controlnet: models/controlnet/
    diffusers: models/diffusers/
    diffusion_models: models/diffusion_models/
    embeddings: models/embeddings/
    gligen: models/gligen/
    hypernetworks: models/hypernetworks/
    LLM: models/LLM/
    llm_gguf: models/llm_gguf/
    loras: models/loras/
    photomaker: models/photomaker/
    style_models: models/style_models/
    unet: models/unet/
    upscale_models: models/upscale_models/
    vae: models/vae/
    vae_approx: models/vae_approx/

**Comment by ltdrdata** [comment]
it seems that your .yaml file is broken.

**Reply by LukeG89** [reply]
@ltdrdata He just mistakenly put the models path (`E:\comfyuimodels\models`) instead of the base path (`E:\comfyuimodels`). Now he fixed it and it works.
